WORKSPACE

```diff
@@ -28,9 +28,9 @@ http_archive(
     patches = [
         "//third_party/io_grpc_grpc_java:054def3c63.patch",
         "//third_party/io_grpc_grpc_java:0959a846c8.patch",
+        "//third_party/io_grpc_grpc_java:1111913510.patch",
```

Comment:
Why changing the order of this patch file? The modification to the content made it compatible with `--incompatible_use_native_patch`, but changing the order of patch files will break applying c63752789e.patch with the following error message:
```
com.google.devtools.build.lib.syntax.EvalException: Error applying patch C:/tools/msys64/home/pcloudy/workspace/bazel-buildfarm/third_party/io_grpc_grpc_java/c63752789e.patch: Incorrect Chunk: the chunk content doesn't match the target
**Original Position**: 1

**Original Content**:
# "repository" here is for Bazel builds that span multiple WORKSPACES.
def _path_ignoring_repository(f):
    if len(f.owner.workspace_root) == 0:

**Revised Content**:
load("@bazel_tools//tools/jdk:toolchain_utils.bzl", "find_java_runtime_toolchain", "find_java_toolchain")

# "repository" here is for Bazel builds that span multiple WORKSPACES.
def _path_ignoring_repository(f):
    if len(f.owner.workspace_root) == 0:
```